app/src/main/java/com/example/myapplication/network/data/UserNews.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.myapplication.network.data
+
+import com.google.gson.annotations.SerializedName
+
+data class UserNews(
```

Comment:
Think about this name and also the name _SingleNews_. It's very important to use names that describe exactly what they are (a good practice is to think what it is/does and name it). For example, if I create a method that receives an array and shrinks it, which name is better? ArrayManager or ArrayShrinker ? ;)